src/constants/task.ts

```diff
@@ -271,3 +279,57 @@ export const mapBadgeColors = {
     text: gray.light3,
   },
 };
+
+export const mapUmbrellaStatusColors = {
```

Comment:
This object will represent all task status color mappings. We still need to confirm with design if we need different colors for the waterfall barchart.